frontend/prisma/schema.prisma

```diff
@@ -12,19 +12,49 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgresql://postgres:postgres@localhost:5555/DATABASE"
```

Comment:
@billwang7599
myb forgot to add this into the readme, but you should copy the `.env.example` file and name it `.env`
that way it'll recognize the "DATABASE_URL" environment variable

if that dosent work lmk